Havit/Havit/Global/UIComponent/ContentsCollectionViewCell.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  ContentsCollectionViewCell.swift
+//  Havit
+//
+//  Created by 박예빈 on 2022/01/12.
+//
+
+import UIKit
+import SnapKit
+import RxSwift
+
+class ContentsCollectionViewCell: UICollectionViewCell {
+    static var cellID = "ContentsCollectionViewCell"
+    
+    var mainImgView: UIImageView!
+    var alarmImgView: UIImageView!
+    var titleLabel: UILabel!
+    var subtitleLabel: UILabel!
+    var dateLabel: UILabel!
+    var linkLabel: UILabel!
+    var moreButton: UIButton!
+    var isReadImgView: UIImageView!
+    var alarmLabel: UILabel!
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setUpCell()
+        setAutoLayouts()
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: .zero)
+        setUpCell()
+        setAutoLayouts()
+    }
+    
+    func setUpCell() {
+        mainImgView = {
+            let imgView = UIImageView()
+            imgView.image = UIImage()
+            
+            return imgView
+        }()
```

Comment:
P3;

위에 UI 컴포넌트 넣어줄때 이런식으로 넣어주면 훨씬 코드가 보기 쉽지 않을까 싶습니다.
```suggestion
       var mainImageView: UIImageView = {
            let imageView = UIImageView()
            imageView.image = UIImage()
            return imageView
        }()
```

Reply:
넵 앞으로는 이렇게 스타일 맞추도록 하겠습니다!